tests/system_tests_user_id.py

```diff
@@ -242,7 +242,7 @@ def create_ssl_domain(self, ssl_options_dict, mode=SSLDomain.MODE_CLIENT):
         domain = SSLDomain(mode)
         if trustfile:
             domain.set_trusted_ca_db(str(trustfile))
-            domain.set_peer_authentication(SSLDomain.VERIFY_PEER, str(trustfile))
+        domain.set_peer_authentication(SSLDomain.VERIFY_PEER, str(trustfile))
```

Comment:
if trustfile is false or None, what does 'str(trustfile)' return?    Should there instead be a "else" to the trustfile test where None is passed as the second arg to set_peer_authentication?

Reply:
Fixed to send in None instead of str(trustfile)